components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -230,6 +233,29 @@ func InjectOAuthProxy(notebook *nbv1.Notebook, oauth OAuthConfig) error {
 	return nil
 }
 
+func (w *NotebookWebhook) ClusterWideProxyIsEnabled() bool {
+	proxyResourceList := &configv1.ProxyList{}
+	err := w.Client.List(context.TODO(), proxyResourceList)
```

Comment:
I'm looking to learn more about the Proxy CR, so far found https://docs.openshift.com/container-platform/4.17/rest_api/config_apis/proxy-config-openshift-io-v1.html. Just a note-to-self.